src/lib/components/scheduling/SongIntersectionCell.svelte

```diff
@@ -0,0 +1,152 @@
+<script lang="ts">
+	import FixedWidthCell from '$lib/components/scheduling/FixedWidthCell.svelte'
+	import type { Musician } from '$lib/types/domain/musician'
+	import {
+		Popover,
+		StructuredList,
+		StructuredListBody,
+		StructuredListCell,
+		StructuredListHead,
+		StructuredListRow
+	} from 'carbon-components-svelte'
+
+	export let position: [number, number]
```

Comment:
ik denk dat het mooier is hier 2 props van te maken genaamd rowIndex en columnIndex